src/makeRobot.js

```diff
@@ -39,6 +40,72 @@
  */
 function makeRobot(name, wheels, version) {
   // write code here
+  const mateRobot = {
+    name: name,
+    'chip version': version,
+    wheels: wheels,
+    coords: {
+      x: 0,
+      y: 0,
+    },
+    get info() {
+      const nameInfo = `name: ${this.name}`;
+      const chipVersionInfo = `chip version: ${this['chip version']}`;
+      const wheelsInfo = `wheels: ${this.wheels}`;
+
+      return `${nameInfo}, ${chipVersionInfo}, ${wheelsInfo}`;
+    },
+
+    goBack(numberOfSteps = 1) {
+      if (numberOfSteps > 0) {
+        this.coords.y -= numberOfSteps;
+      }
+
+      return this;
+    },
+
+    goForward(numberOfSteps = 1) {
+      if (numberOfSteps > 0) {
+        this.coords.y += numberOfSteps;
+      }
+
+      return this;
+    },
+
+    goLeft(numberOfSteps = 1) {
+      if (numberOfSteps > 0) {
+        this.coords.x -= numberOfSteps;
+      }
+
+      return this;
+    },
+
+    goRight(numberOfSteps = 1) {
```

Comment:
make it simple, everywhere
```suggestion
    goRight(steps = 1) {
```